include/camera/camera.h

```diff
@@ -0,0 +1,158 @@
+#pragma once
+#include <wut.h>
+
+/**
+ * \defgroup cam Camera
+ * \ingroup cam
+ * @{
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define CAMERA_WIDTH 640
+#define CAMERA_PITCH 768
+#define CAMERA_HEIGHT 480
+
+#define CAMERA_Y_BUFFER_SIZE  (CAMERA_PITCH * CAMERA_HEIGHT)
+#define CAMERA_UV_BUFFER_SIZE (CAMERA_PITCH * CAMERA_HEIGHT / 2)
+#define CAMERA_YUV_BUFFER_SIZE (CAMERA_Y_BUFFER_SIZE + CAMERA_UV_BUFFER_SIZE)
+
+#define CAMERA_YUV_BUFFER_ALIGNMENT 256
+
+#define CAM_THREAD_AFFINITY_ANY 0x00000007u
+
+typedef int CAMHandle;
+typedef int CAMError;
+
+typedef struct _Camera_Event_Handler_Input _Camera_Event_Handler_Input;
+typedef struct CAMMode CAMMode;
+typedef struct CAMWorkMem CAMWorkMem;
+typedef struct CAMStreamInfo CAMStreamInfo;
+typedef struct CAMSetupInfo CAMSetupInfo;
+typedef struct CAMSurface CAMSurface;
+
+typedef enum CamError
+{
+   CAMERA_ERROR_OK                     =  0,
+   CAMERA_ERROR_INVALID_ARG            = -1,
+   CAMERA_ERROR_INVALID_HANDLE         = -2,
+   CAMERA_ERROR_INSUFFICIENT_MEMORY    = -5,
+   CAMERA_ERROR_NOT_READY              = -6,
+   CAMERA_ERROR_UNIINITIALIZED         = -8,
+   CAMERA_ERROR_UNKNOWN                = -10,
+   CAMERA_ERROR_DEVICE_IN_USE          = -12,
+   CAMERA_ERROR_SEGMENT_VIOLATION      = -14
+} CamError;
+
+typedef enum CamFps
+{
+   CAMERA_FPS_15,
+   CAMERA_FPS_30
+} CamFps;
+
+typedef enum CamStreamType
+{
+   CAMERA_STREAM_TYPE_1
+} CamStreamType;
+
+struct _Camera_Event_Handler_Input
```

Comment:
This naming seems weird to me. Thoughts about naming it something like `CAMEvent` or `CAMEventData`?